apps/drec-ui/src/organization/logic/invitations/sentInvitationsTable.ts

```diff
@@ -0,0 +1,25 @@
+import { InvitationDTO } from '@energyweb/origin-drec-api-client';
+import { TableComponentProps } from '@energyweb/origin-ui-core';
+
+const prepareSentInvitation = (invite: InvitationDTO) => ({
+    id: invite.id,
+    email: invite.email,
+    status: invite.status
+});
+
+export const useSentInvitationsTableLogic = (
+    invitations: InvitationDTO[],
+    loading: boolean
+): TableComponentProps<InvitationDTO['id']> => {
+    return {
+        header: {
+            email: 'Email',
+            status: 'Status'
+        },
+        loading,
+        pageSize: 5,
+        totalPages: Math.ceil(invitations?.length / 5),
```

Comment:
I think this property could be removed, since Table component should have its own page calculator func.